charts/cluster-scanner/values.yaml

```
@@ -0,0 +1,124 @@
+#basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  sysdigHost: ""
+  sysdigAccessKey: ""
+  clusterName: ""
+  sysdigSkipVerifyCertificate: false
+
+  scannerMode: "local" # or "multi"
+  loggingLevel: "INFO"
+
+eveEnabled: false
+eveIntegrationEnabled: false
+rootNamespace: "kube-system"
+
+multicluster:
+  # kubeconfigSecretName: ""
+
+localcluster:
```

We are using camelCAse for all the other config, I suggest to keep consistency.

```suggestion
localCluster:
```

---

Seems not used, can we remove it?

---

it's actually a placeholder: the current service implementation does not support it, but it will

cc @michele-mangili

---

We have finalized the implementation and the `localCluster` config is being used. See https://github.com/sysdiglabs/charts/blob/80bb5bbbde270249bc4bb7bca5413e49e117bc1b/charts/cluster-scanner/values.yaml#L84